### PR TITLE
{Release} Upgrade to Azure CLI 2.56.0

### PR DESCRIPTION
**Pull request by azclibot**
Updated Azure CLI version.

### CODE REVIEW

**Comment by yonzhan** [comment]
Release

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,112 @@
...
+* `az aks trustedaccess rolebinding list`: Add new command to list all the trusted access role bindings (#27931)
+* `az aks trustedaccess rolebinding show`: Add new command to get the specific trusted access role binding according to binding name (#27931)
+* `az aks trustedaccess rolebinding create`: Add new command to create a new trusted access role binding (#27931)
+* `az aks trustedaccess rolebinding update`: Add new command to update an existing trusted access role binding (#27931)
+* `az aks trustedaccess rolebinding delete`: Add new command to delete a trusted access role binding according to name (#27931)
+* `az aks update`: Add parameter `--network-plugin` to update the network plugin of the AKS cluster (#28042)
+
+**App Config**
+
+* `appconfig feature set`: Add requirement type to feature command (#28065)
```
Comment (evelyn-ys) [inline src/azure-cli/HISTORY.rst:27]:
```suggestion
* `az appconfig feature set`: Add requirement type to feature command (#28065)
```

File: src/azure-cli/HISTORY.rst [hunk 2]
```diff
@@ -3,6 +3,112 @@
...
+* `az network nsg rule list`: Fix ranges and prefixes disappeared with `-o table` option (#27972)
+* `az network vnet subnet`: Add `--private-endpoint-network-policies` and `--private-link-service-network-policies` (#28023)
+
+**RDBMS**
+
+* `az postgres flexible-server virtual-endpoint`: Add support for virtual endpoints for PostgreSQL flexible server (#27885)
+* `az postgres flexible-server replica promote`: Add capability to stop replication and promote to primary or standalone server with the selection of planned/force data syncs. (#28013)
+* `az postgres flexible-server server-logs list`: List server log files for PostgreSQL flexible server (#28020)
+* `az postgres flexible-server server-logs download`: Download server log files for PostgreSQL flexible server (#28020)
+* Add support for provisioning and updating Premium SSDV2 Disks in CLI for PostgreSQL flexible servers (#28039)
```
Comment (evelyn-ys) [inline src/azure-cli/HISTORY.rst:93]:
```suggestion
* [RDBMS] az postgres flexible-server create: Add capability to set storage type to PremiumV2_LRS and provide values for IOPS and Throughput during creation (#28039)
* [RDBMS] az postgres flexible-server update: Add capability to update the values of IOPS and Throughput during update (#28039)
```

File: src/azure-cli/HISTORY.rst [hunk 3]
```diff
@@ -3,6 +3,113 @@
...
+* `az network vnet subnet`: Add `--private-endpoint-network-policies` and `--private-link-service-network-policies` (#28023)
+
+**RDBMS**
+
+* `az postgres flexible-server virtual-endpoint`: Add support for virtual endpoints for PostgreSQL flexible server (#27885)
+* `az postgres flexible-server replica promote`: Add capability to stop replication and promote to primary or standalone server with the selection of planned/force data syncs. (#28013)
+* `az postgres flexible-server server-logs list`: List server log files for PostgreSQL flexible server (#28020)
+* `az postgres flexible-server server-logs download`: Download server log files for PostgreSQL flexible server (#28020)
+* [RDBMS] az postgres flexible-server create: Add capability to set storage type to PremiumV2_LRS and provide values for IOPS and Throughput during creation (#28039)
+* [RDBMS] az postgres flexible-server update: Add capability to update the values of IOPS and Throughput during update (#28039)
```
Comment (evelyn-ys) [inline src/azure-cli/HISTORY.rst:94]:
```suggestion
* `az postgres flexible-server create`: Add capability to set storage type to PremiumV2_LRS and provide values for IOPS and Throughput during creation (#28039)
* `az postgres flexible-server update`: Add capability to update the values of IOPS and Throughput during update (#28039)
```

**Review by evelyn-ys — APPROVED** [review]

**Review by necusjz — APPROVED** [review]

**Review by AllyW — APPROVED** [review]

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**